controllers/localization_controller.go

```diff
@@ -83,11 +83,15 @@ func (r *LocalizationReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 	if err := mgr.GetFieldIndexer().IndexField(context.TODO(), &v1alpha1.Localization{}, configKey, func(rawObj client.Object) []string {
 		loc := rawObj.(*v1alpha1.Localization)
-		var ns = loc.Spec.ConfigRef.Namespace
```

Comment:
Adopting this change in https://github.com/open-component-model/ocm-controller/pull/250 so Gerald can't move forward with some demos.